src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -458,7 +459,7 @@ def makeJobSubmit(self, task):
         info['runs'] = []
         info['lumis'] = []
         info['saveoutput'] = 1 if info['tm_transfer_outputs'] == 'T' else 0
-        if info['userhn'] in getattr(self.config.TaskWorker, 'highPrioUsers', []):
+        if info['userhn'] in self.getHighPrioUsers():
```

Comment:
Should probably not fail submission and (maybe?) add a warning https://github.com/dmwm/CRABServer/blob/master/src/python/TaskWorker/Actions/DBSDataDiscovery.py#L43